wsagent/che-datasource-api/pom.xml

```diff
@@ -0,0 +1,179 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <artifactId>che-agent-parent</artifactId>
+        <groupId>org.eclipse.che.core</groupId>
+        <version>5.15.0-SNAPSHOT</version>
+    </parent>
+    <artifactId>che-datasource-api</artifactId>
+    <packaging>jar</packaging>
+    <name>CHE :: Datasource :: API</name>
+    <properties>
+        <dto-generator-out-directory>${project.build.directory}/generated-sources/dto/</dto-generator-out-directory>
+    </properties>
+    <dependencies>
+        <dependency>
+            <groupId>com.google.code.gson</groupId>
+            <artifactId>gson</artifactId>
+            <version>2.3.1</version>
```

Comment:
all versions should be removed from this dependencies. They are all should come from parent pom